README.md

```diff
@@ -50,7 +50,7 @@ BD::union()->all();
 
 To get all the districts of division that start with 3 character `chi`(Chittagong), we can use the following code. Same thing will work for sub-disctrict and union. But in case of sub-district and union we need to pass `district` and `sub_district` respectively.  
 ```
-BD::district()->getWhere('division', 'chi');
+BD::district()->getWhere('division', 'cha');
```

Comment:
Thanks for this fix but I think you have missed the same mistake in the paragraph before this code block. Can you please fix those spelling also?